audio/abcde/Portfile

```diff
@@ -1,9 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+PortGroup               perl5 1.0
 
 name                    abcde
-version                 2.8.1
+version                 2.9.1
+revision                1
```

Comment:
This shouldn't be `revision 1` since you're bumping the `version` in this commit

Reply:
This is currently two separate commits: ffeb904 bumps version to 2.9.1, 24c8338 bumps revision to 1. (GitHub's "Files changed" tab shows all commits together as if they were one commit.)

Reply:
So what? They'll be merged as a group, so there won't be some point where 2.9.1 is available to users without the rest. (And in fact, you should probably squash these...)